run-smoke-test.sh

```diff
@@ -5,4 +5,14 @@ rm ember-try*.tgz || true
 npm pack
 now=$(date +%s)
 mv ember-try*.tgz "ember-try-${now}.tgz"
-cd smoke-test-app && yarn upgrade "ember-try@file:../ember-try-${now}.tgz" && yarn install && "./${1}"
+cd smoke-test-app
+
+if [ "$2" == "no-yarn" ]; then
+  npm i
+  npm i --save-dev "../"
```

Comment:
Why wouldn't the `npm` smoke test also use the tarball?

Reply:
So it seems like this was a bit of a red herring 🤔 I originally thought it was the installing via the tarball that was causing the issue but in fact it was having it missing from the lock file that was the problem. I will make it so npm also uses the tarball 👍